.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+bun.lockb
```

Comment:
Use a global gitignore for something like this https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Yeah I just undid this.